### ISSUE TITLE
Dicty phenotype : wild type

### ISSUE THREAD

**Issue by pfey03** [obophenotype/upheno#758]
When researchers investigate developmental and / or cellular phenotypes and find no difference to parental strain, we annotate this as DDPHENO:0000142 wild type.

We have over 4,000 thus annotations and users find them valuable info as they come with distinct references.

Do others annotate when no phenotype has been detected with the methods used in the reference? 
If yes, how do others annotate these?

Maybe a topic for a phenobarbitals call, thanks!

**Comment by matentzn** [comment]
Yes, this is an issue for the call - and one of the largest headaches. Most resources assume that "no phenotype" means "wild type"; but for a huge chunk of the data in MODs, that is simply untrue. In MGI, for example, "no phenotype" or "normal phenotype" means "no phenotype compared to control". The solution to this problem is to get sources to release the "comparator" of the phenotypic effect - when no comparator is given we assume no/rmal means wild. I actually kinda like "wild type" much more than "normal phenotype" - because it makes that explicit! Lets have a good discussion.

**Comment by pfey03** [comment]
Yes, should be a good discussion. We typically have a note that provides information about which phenotypes were checked and the linked reference should give more info. The parent strain will also be linked in our emerging database. 
Example: https://tinyurl.com/3n3b4kmb

**Comment by mah11** [comment]
I tend to bang on to the effect that there's no such thing as "no phenotype" - every living thing has a phenotype! Anyway, we all know "no phenotype" is just a lazy way of saying "no difference from control (which is usually wild type)", so moving on ....

FYPO has a big "normal phenotype" branch (see FYPO:0000257), with a lot of terms that allow us to specify which process, structure, etc. is normal. All the text definitions expliticly say normal = indistinguishable from wild type. It also makes our lives easier that the vast majority of FYPO usage is for S. pombe, and the vast majority of S. pombe experiments use the same strain background as "wild type".

We provide instructions for annotators that includes this blurb about "normal":

> Normal phenotypes
> 
> Often, a mutant will be assayed for a phenotype but will appear identical to wild-type cells in the assay. One or more of the "normal phenotype" terms, which are defined as having no observable difference from wild type under the given conditions, can be used to annotate these mutants. For example, an allele may be annotated to "normal growth on amphotericin B" or "normal cell morphology". A mutant may be "normal" in one respect, but abnormal in another (for instance, cells may grow at a normal rate, but have an abnormal morphology). Note that it is not necessary to annotate all respects in which a mutant cell is normal, only those that are unexpected or otherwise interesting.

**Comment by pfey03** [comment]
@mah11 how to you deal with the 'normal' qualifier in UPHENO? 
I prefer 'wild type' as the phenotype referring to the investigated aspects, for Dicty typically growth and development, compared to the parent. I would have to inspects some of ours closer, but we have thousands and I hope there will be an easy solution.

**Comment by sbello** [comment]
The MP also has a branch for normal phenotype (MP:0002873) which has the direct children "no abnormal phenotype detected" (http://purl.obolibrary.org/obo/MP_0002169) and "phenotypic reversion" 
"no abnormal phenotype detected" is what we use for most annotations when there is no significant difference in phenotype compared to control.
The MP also has a term for "no phenotypic analysis" which we had been using to keep track of papers that had a new allele but no phenotype for that allele. This term doesn't really belong in the ontology and we are handling these in a different way now and slowly cleaning up the existing annotations. Eventually this term will be obsoleted.

**Comment by mah11** [comment]
> how to you deal with the 'normal' qualifier in UPHENO?

That's probably a question for @matentzn or @dosumis or ?? ... at least, I don't think I have a good answer yet. FYPO logical definitions follow a de-facto pattern that uses a 'normal' qualifier that's analogous to 'abnormal, but 'normal yada' patterns haven't made it into the uPheno collection yet :/ 

FYPO LDs may end up changing when uPheno does have patterns for normal/wild type/whatever.

> I prefer 'wild type' ...

Because we think of (and define) 'normal' as "same as wild type" for fission yeast, I don't have a preference either way (beyond the small thing that leaving our term names as "normal" will save some text editing).

I wouldn't call a cell "wild type" unless all of its attributes are normal, whereas FYPO has a lot of 'normal X' phenotype terms that are defined as "X is normal (but implicitly, Y or Z may be normal or abnormal)". But for each attribute, "normal thing" still means "this thing is the same as in a completely wild-type cell" (and of course, cell = organism for us).

> I hope there will be an easy solution.

Me too, but I suspect that if it were easy, we wouldn't still be working on it!

**Comment by pfey03** [comment]
We have nearly 3,000 annotations to wild type. Just starting to inspect, we also have rescued mutants annotated with 'wild type'. 
We annotate phenotypes for nearly 20 years and we have a Stock Center, which was the first years located at Columbia University. Therefore we had a lot of people annotating phenotypes early on and put details what phenotypes have been checked, into free text notes.

A note like this: "rescues actin incorporation into the cytoskeleton, pseudopod formation, cel motility and morphology, phagocytosis, and chemotaxis defects"

This would be a lot of 'normal phenotypes'.
